number-46/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+    <script>
+      class Shape {
+        constructor(length, width) {
+          this.length = length;
+          this.width = width;
+        }
+
+        get area() {
+          return this.length * this.width;
+        }
+
+        get name() {
+          return "Default Shape";
+        }
+      }
+
+      class Square extends Shape {
+        constructor(length) {
+          super(length, length);
+        }
+
+        get name() {
+          return "Square";
+        }
+      }
+
+      class Rectangle extends Shape {
+        constructor(length, width) {
```

Comment:
Since this `constructor` is exactly identical as Shape's `constructor`, it is not necessary. It will automatically take everything inside the `Shape` class.